### PR TITLE
Replace bitmap blit with ngfx blit

### PR DESCRIPTION
**Pull request by Helco**
So this turned out to be quite a bit of work, I'll try to summarize the issues on the way: 

- png loader: did not set `free_data_on_destroy`, thus the aggressive loading of the menu filled up RAM very fast
- png loader: the no-palette-but-alphas fix for 4Bit is also needed for 2Bit/1Bit palette bitmaps 
- png loader: 1Bit uses lsb->msb bitorder, for now I opted for creating a palette (like with ^) and turning it into 1BitPalette
- app_manager: `rwatch_neographics_init` was being called quite often, dicussed temporary fix for this was moving the call into the overlay thread init (tracked in pebble-dev/RebbleOS#62)
- menu: should set the compop to `Set` before drawing icons as to enable blending

### CODE REVIEW

File: lib/png/png.c [hunk 1]
```diff
@@ -40,6 +42,8 @@ void png_to_gbitmap(GBitmap *bitmap, uint8_t *raw_buffer, size_t png_size)
         goto freepng;
     }
 
+    SYS_LOG("png", APP_LOG_LEVEL_INFO, "UPNG Decode format: %d", upng_get_format(upng));
```
Comment (jwise) [inline lib/png/png.c]: debug spew unnecessary?
```suggestion
```

**Comment by jwise** [comment]
Looks great to me.  Does this depend on #130 and the resource change associated?  If not, feel free to merge.

**Comment by Helco** [comment]
Yes, Tintin won't show any bitmaps without #130

**Comment by jwise** [comment]
Ok, add one more commit to this PR to update the submodule and then I'll merge the whole thing?

**Comment by Helco** [comment]
sure there you go

**Comment by jwise** [comment]
:tada:

**Pull request merged**